docs/UserGuide.md

```diff
@@ -2,53 +2,43 @@
-
-### About this guide
-This guide provides a basic introduction for using TutorBuddy, and a detailed description for each feature.<br/>
-Go to [Quick start](#quick-start), for a summary of step to get started with using the application.<br/>
-Go to [Features](#features), for a detailed explanation of each feature. <br/>
-Go to [FAQ](#faq), for answers to Frequently Asked Questions. <br/>
-Go to [Command Summary](#command-summary), for a basic summary to commands.
-
 **Table of Contents**
+* [Introduction](#introduction)
 * [About](#about)
 * [Quick start](#quick-start)
-* [Features](#features)
-  * [Managing Students Commands](#managing-students-commands)
-    * [Listing all students: `list_student`](#listing-all-students-list_student)
-    * [Locating student profile by name: `find_student`](#locating-student-profile-by-name-find_student)
+* [Commands](#commands)
+  * [General](#general)
+    * [Open help panel: `help`](#viewing-help--help)
+    * [Listing all students and sessions: `list_student`](#listing-all-students-and-sessions-list_student)
+    * [Clearing all data : `clear`](#clearing-all-data--clear)
+    * [Exit the program: `exit`](#exit-the-program-exit)
+  * [Managing Students](#managing-students)
     * [Adding a student: `add_student`](#adding-a-student-add_student)
-    * [Editing a student: `edit_student`](#editing-a-student-edit_student)
+    * [Locating student by name: `find_student`](#locating-student-by-name-find_student)
     * [Deleting a student: `delete_student`](#deleting-a-student-delete_student)
-  * [Managing Sessions Commands](#managing-sessions-commands)
-    * [Listing all tuition sessions: `list_session`](#listing-all-tuition-sessions-list_session)
-  <!--* [Locating tuition session by student name / date: `find_session`](#locating-tuition-session-by-student-name--date-find_session)-->
+    * [Editing a student: `edit_student`](#editing-a-student-edit_student)
+    * [Listing students' emails based on current list: `emails`](#listing-students-emails-based-on-current-list-emails)
+  * [Managing Sessions](#managing-sessions)
     * [Adding a tuition session: `add_session`](#adding-a-single-tuition-session-add_session)
     * [Adding a recurring tuition session: `add_rec_session`](#adding-a-recurring-tuition-session-add_rec_session)
     * [Deleting a tuition session: `delete_session`](#deleting-a-tuition-session-delete_session)
-  * [Miscellaneous Commands](#miscellaneous-commands)
-    * [Open help panel: `help`](#viewing-help--help)
-    * [Listing students' emails based on current list: `emails`](#listing-students-emails-based-on-current-list-emails)
+  * [Fees](#fees)
     * [Getting monthly fee for a particular student: `fee`](#getting-monthly-fee-for-a-particular-student-fee)
-    * [Clears all entries in the program: `clear`](#clearing-all-entries--clear)
-    * [Exit the program: `exit`](#exit-the-program-exit)
-* [FAQ](#faq)
 * [Command Summary](#command-summary)
+* [FAQ](#faq)
+* [Glossary](#glossary)
```

Comment:
Nice! Was thinking if it might be good to use <div style="page-break-after: always;"></div> as stated in https://se-education.org/guides/tutorials/savingPdf.html for us to ensure that the introduction section always starts at a new page. Might be good to include this at before the start of a key section (E.g. Before Intro, Commands and Command Summary)

Reply:
Added!